exchanges/exchange.go

```diff
@@ -1943,3 +1944,37 @@ func (b *Base) Bootstrap(_ context.Context) (continueBootstrap bool, err error)
 func (b *Base) IsVerbose() bool {
 	return b.Verbose
 }
+
+// GetDefaultConfig returns a default exchange config
+func (b *Base) GetDefaultConfig(ctx context.Context, instance LimitedScope) (*config.Exchange, error) {
```

Comment:
I agree with your PR assessment that its a bit jank. I think this can become an exported function instead
`func GetDefaultConfig(ctx context.Context, instance IBotExchange) (*config.Exchange, error) {`
That or `LimitedScope`, but I don't see much problem with using IBotExchange